app/models/report/standard_form_report.rb

```diff
@@ -89,16 +89,38 @@ def run(current_ability)
   end
 
   # returns all non-admin users in the form's mission with the given role that have not submitted any responses to the form
-  # options[:role] - (symbol) the role to check for
+  # options[:role]) the role to check for
+  # options[:limit]) how many users we want to fetch from the db
   def users_without_responses(options)
-    # eager load responses with users
-    all_observers = form.mission.assignments.includes(:user).find_all{|a| a.role.to_sym == options[:role] && !a.user.admin?}.map(&:user)
-    submitters = form.responses.includes(:user).map(&:user).uniq
-    @users_without_responses = all_observers - submitters
+    User.without_responses_for_form(form, options).sort
   end
 
-  def observers_without_responses
-    users_without_responses(:role => :observer)
+  # This returns the count of the previous users select without the limit clause.
+  # Format: [[count]]
+  def observers_without_responses_count
+    ActiveRecord::Base.connection.execute('SELECT FOUND_ROWS();').entries[0].first
+  end
+
+
+  def observers_without_responses(limit=100)
+    users = users_without_responses({role: :observer, limit: limit})
+
+    if users.empty?
+      I18n.t('report/report.zero_missing_observers')
+    else
+      format_observers_names(users, limit)
+    end
+  end
+
+  def format_observers_names(users, limit)
+    users_list = users.map{|u| u.name }.join(', ')
+
+    users_total_count = observers_without_responses_count
+    total_count_label = I18n.t('report/report.missing_observers_total_count',
+                             {user_count: users_total_count})
```

Comment:
Just one level of indent please. Also you don't need the enclosing curly braces here.